AutomatedTesting/AutomatedTesting_Dependencies.xml

```diff
@@ -1,5 +1,4 @@
 <EngineDependencies versionnumber="1.0.0">
     <Dependency path="libs/particles/preloadlibs.txt" optional="true" />
     <Dependency path="libs/gameaudio/wwise/*.xml" optional="false" />
-    <Dependency path=":libs/gameaudio/wwise/levels/default_controls.xml" optional="false" />
```

Comment:
I think this {ProjectName}_Dependencies.xml used to be part of the new project template. Does this need to be updated there? I can't find it in the template folder.

Reply:
That I don't know.  The contents of this do seem (somewhat) specific to AutomatedTesting.  I was removing the duplication here.  Not all projects have `libs/gameaudio/wwise/*.xml`.  Maybe make it `optional=true` would be fine.

Reply:
If anything theses dependencies are project specific.

By default a new project would not have any dependencies within it.

Reply:
> If anything theses dependencies are project specific. By default a new project would not have any dependencies within it.

In Lumberyard at least there were hardcoded C++ loads to assets with these paths, which is why this would be in the project template, as {ProjectName}_Dependencies.xml. Without this file, if you attempted to bundle your game and if you had wwise audio XML files or particle system preloadlibs.txt, they wouldn't get packaged and your bundled release build would be missing content.